src/controller/CHIPDeviceController.h

```diff
@@ -351,6 +358,20 @@ class DLL_EXPORT DeviceController : public AbstractDnssdDiscoveryController
      */
     CHIP_ERROR InitControllerNOCChain(const ControllerInitParams & params);
 
+    /**
+     * @brief Update the NOC chain of controller.
+     *
+     * @param[in] noc                                NOC in CHIP certificate format.
+     * @param[in] icac                               ICAC in CHIP certificate format.
```

Comment:
Oh, and my question here about "What should get passed in if there is no ICAC?" means: the API documentation should document that.

Reply:
Ok.